docs/api.md

```diff
@@ -40,6 +40,8 @@
 .. autofunction:: scores.probability.crps_step_threshold_weight
 .. autofunction:: scores.probability.crps_cdf_brier_decomposition
 .. autofunction:: scores.probability.crps_for_ensemble
+.. autofunction:: scores.probability.twcrps_for_ensemble
```

Comment:
I wonder if this should be tw_crps_for_ensemble.

Reply:
I'm in two minds.
1. The other threshold weighted scores are `tw_...`
2. In the literature it's usually written as twCRPS. We use the acronym in the threhsold weighted CRPS case, so it perhaps makes sense to drop the underscore, but for the other threshold weighted scores we don't use an acronym (e.g tw_squared_error rather than twse)

I'm okay either way and am happy to go with what people think is clearest.

Reply:
Thanks for the explanation. Based on that, I think either way is acceptable, but I personally lean towards including the underscore.

Reply:
I started updating this, but felt that it started looking messy with the difference between the function name and referring to twCRPS in docs. I'll leave it as is unless you (or anyone else) has a stronger opinion in which case, I will update it.

Reply:
I gathered a few more options, we have a general concensus to prefer tw_crps and including the underscore for consistency across the codebase (two in favour of the underscore, nil against)

Reply:
Okay - thanks for checking with others. I'll update that now.